lnwallet/wallet.go

```diff
@@ -1131,6 +1151,16 @@ func (l *LightningWallet) CheckReservedValueTx(req CheckReservedValueTxReq) (
 	return reservedVal, nil
 }
 
+// ReservedValue returns the total value that must be reserved given the number
+// of anchor channels.
+func ReservedValue(numAnchorChans int) btcutil.Amount {
```

Comment:
In the meantime we have a method for this already, check `l.RequiredReserve(numChannels)`.